regression/cbmc/stack-trace/test.desc

```diff
@@ -0,0 +1,11 @@
+CORE
+main.c
+--stack-trace
+^EXIT=10$
+^SIGNAL=0$
+^  assertion failure file main.c line 3 .*$
```

Comment:
Use `activate-multi-line-match` to check the right order of functions

Reply:
ok, done